ui/src/app/kvm/components/KVMHeaderForms/DeleteForm/DeleteForm.tsx

```diff
@@ -90,7 +101,17 @@ const DeleteForm = ({
           );
         }
       }}
+      onSuccess={() => {
+        dispatch(
+          messageActions.add(
+            `${pod ? "KVM" : "Cluster"} removed successfully`,
+            NotificationSeverity.INFORMATION
+          )
+        );
+        history.push({ pathname: kvmURLs.kvm });
+      }}
       processingCount={deletingCount}
+      saved={deleted}
```

Comment:
We should also make sure the delete action was successful, i.e. there are no errors

Reply:
I've added this (and a bunch of tests). Would you mind taking another look? The changes are in the second commit.